openrtb_ext/imp_nextmillennium.go

```diff
@@ -1,5 +1,6 @@
 package openrtb_ext
 
 type ImpExtNextMillennium struct {
-	PlacementID string `json:"placement_id"`
+	GroupID     *string `json:"group_id,omitempty"`
```

Comment:
Could you explain why you made this new `GroupID` value a pointer to a string?

Reply:
Good point, @AlexBVolcy. I think you'd be fine with a `string` instead of `*string`.

Reply:
Done!